truss/templates/triton/model/triton_model_wrapper.py

```diff
@@ -0,0 +1,80 @@
+import importlib
+import inspect
+import sys
+from pathlib import Path
+from typing import Dict
+
+import triton_python_backend_utils as pb_utils
+from shared.secrets_resolver import SecretsResolver
+from transform import transform_pydantic_to_triton, transform_triton_to_pydantic
+from utils.pydantic import inspect_pydantic_model
+from utils.signature import signature_accepts_keyword_arg
+
+
+class ModelWrapper:
```

Comment:
nit: let's call this TritonModelWrapper to:
1. Match the name of the file
2. Not to be confused with a different ModelWrapper class that we have in Truss